SuperTeamPicker/routes/cohorts.js

```diff
@@ -0,0 +1,109 @@
+const path = require('path');
+const multer = require('multer');
+const express = require('express');
+const router = express.Router();
+const knex = require('../db');
+
+const UPLOADS_DIR = 'uploads';
+const upload = multer({dest: path.join(__dirname, '..', 'public', UPLOADS_DIR)});
+
+// PATH: /cohorts/new VERB: GET Serves form for creating cohorts
+router.get('/new', (request, response) => {
+  // response.send('TEST POSTS NEW')
```

Comment:
Just as an fyi it's good industry practice to remove unused, commented out code as part of a clean up step/commit before creating a PR. It's not a big deal at this stage, but just something to keep in mind.